apps/indexer/src/resolvers/url-resolver.ts

```diff
@@ -54,10 +56,14 @@ async function resolveURL(
   }
 
   if (contentType.startsWith("image/")) {
+    const dimension = await getImageDimension(response, abortController).catch(
+      () => undefined,
+    );
     return {
       type: "image",
       mediaType: contentType,
```

Comment:
size `image-size` is able to return real media type it detected, could we use that instead?

Reply:
i've updated the module to do so, `image-size` actually isn't returning a full media type, so had to map it and fallback to contentType